docs/Tutorial_VALMOD.ipynb

```diff
@@ -0,0 +1,1137 @@
+{
```

Comment:
<p>Performing the expansion here feels very disconnected from equation (3)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/586/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Reply:
I can feel it now. So, I will change the structure. I will start expanding (3) right after eq(3). THEN, I can show the partial derivative  of  function w.r.t. to the other variable (i.e. `\sigma_prime`) followed by its expansion. I will revise  it in the next set of commits.

Reply:
<p>Yes. Thanks</p>